Assets/Scripts/ARFaceArkitBlendShapeVisualizer.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Experimental.XR;
+using UnityEngine.XR.ARFoundation;
+using UnityEngine.XR.ARKit;
+
+/// <summary>
+/// Populates the action unit coefficients for an <see cref="ARFace"/>.
+/// </summary>
+/// <remarks>
+/// If this <c>GameObject</c> has a <c>SkinnedMeshRenderer</c>,
+/// this component will generate the blend shape coefficients from the underlying <c>ARFace</c>.
+/// 
+/// </remarks>
+[RequireComponent(typeof(ARFace))]
+public class ARFaceArkitBlendShapeVisualizer : MonoBehaviour
+{
+    [SerializeField]
+    float coefficientScale = 100.0f;
+    [SerializeField]
+    SkinnedMeshRenderer skinnedMeshRenderer = null;

```

Comment:
```suggestion
    SkinnedMeshRenderer m_SkinnedMeshRenderer;
```

`null` is already the default value here (and explicitly setting it is against our coding convention). Why not give it public accessors?

The setter would also need to regenerate the `m_FaceArkitBlendShapeIndexMap`.